data/coalition/coalition missions.txt

```diff
@@ -1393,3 +1393,58 @@ mission "Arachi Cartoonist 2"
+			choice
+				`	(Stay.)`
+					goto end
+				`	(Leave.)`
+			label leaveearly
+			`	You thank the Arach beside you for inviting you here, and while she doesn't seem to pay you much mind, you can feel the gazes of other Arachi patrons on you, whispering among themselves as you leave. A few minutes after leaving, you're back at your ship.`
+				decline
+			label end
+			`	The theater workers leave one after the other, leaving you and the other guests with an empty stage. After a while, the lights are turned off. About ten minutes later, however, you hear the curtains open, as some violins start to play in unison, not from where the orchestra was seated, but as if they were all over the theater, behind you and even up on the balconies. A strong light comes from the stage, most of it blocked by a tall, dark figure, but it isn't an actor, seemingly being made of some sort of wooden parts, kept up by thin wires you can barely spot when looking at the top of the stage. The violins' song is disrupted by dull, successive drum hits, at which point the figure moves, with "arms" opening to the sides, and becoming even taller, as it stands up. It is a humanoid creature, and it begins to slowly move around the stage, as if dancing in a pendulum. Cellos join in with the violin screams, and are followed by a choir, though you cannot understand what they said. You do recognize, however, that it isn't being sung in any language of the Coalition.`
+			`	The creature appears to get closer to the audience, growing in size, and as the choir and instruments grown more intense, it swings one of its arms upward, removing the wires. It stays upright, however, and continues to move, now bring both arms close, and striking its own head, forming two small holes from which the background light shines through. It falls to its knees, with one arm reaching for the audience, as the light behind it turns off, and the choir and music subside.`
+			`	The lights turn back on, revealing curtains still open, but nothing on the stage. A few workers come in to tidy up the place, and the remaining guests now get up to leave. "Enjoyed the soiree, have you, Captain?" The Arach beside you asks, as she herself gets up to leave. "Come now, actually over, the show now is." You follow her outside, and she guides you back to your ship. "Thankful for you accompanying me, I am, Captain. Safe travels, may you have."`
```

Comment:
I think the player should have the chance to ask what the heck it was they just saw. To be honest, there doesn't even have to be a substantial answer, or you can have the Arach avoid the question, but I would say let the player at least ask. That's a really strange experience; it would feel weird to just accept and move on with your day.
Also - what's the figure actually supposed to be? Some strange cultural thing? Foreshadowing for something in the Heliarch/Resistance campaigns? Just really curious about this myself.

Reply:
The figure is meant to be a Quarg, and the choir is in a language the player doesn't recognize for it is sung in the Quarg language.
After the events of the Quarg war, the Coalition would've gotten a boogeyman story of sorts related to the Quarg, this was an Arach take on the tale, being one of the plays.

I'll try and add a thing later about the player trying to ask about it.